downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -0,0 +1,177 @@
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
+* `automationhub_ldap_bind_dn`
+* `automationhub_ldap_bind_password`
+* `automationhub_ldap_user_search_base_dn`
+* `automationhub_ldap_group_search_base_dn`
+
+If any of these variables are missing, the Ansible Automation installer will  not complete the installation. 
+
+
+== Setting up your inventory file variables
+
+When you configure your {PrivateHubName} with LDAP authentication, you must set the proper variables in your inventory files during the installation process.
+
+//.Prerequisites (Add Prereqs to 2.2 only)
+
+//* Ensure that your system is running {PlatformName} 2.2.1 or later.
+//* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
+
+. Use the following example as a guide to set up your {PlatformNameShort} inventory file:
++
+-----
+automationhub_authentication_backend = "ldap"
+
+automationhub_ldap_server_uri = "ldap://ldap:389"   (for LDAPs use  automationhub_ldap_server_uri = "ldaps://ldap-server-fqdn")
+automationhub_ldap_bind_dn = "cn=admin,dc=ansible,dc=com"
+automationhub_ldap_bind_password = "GoodNewsEveryone"
+automationhub_ldap_user_search_base_dn = "ou=people,dc=ansible,dc=com"
+automationhub_ldap_group_search_base_dn = "ou=people,dc=ansible,dc=com"
+-----
++
+[Note] 
+====
+The following variables will be set with default values, unless you set them with other options.
+
+-----
+auth_ldap_user_search_scope= `SUBTREE'
+auth_ldap_user_search_filter= `(uid=%(user)s)`
+auth_ldap_group_search_scope= 'SUBTREE'
+auth_ldap_group_search_filter= '(objectClass=Group)`
+auth_ldap_group_type_class= 'django_auth_ldap.config:GroupOfNamesType'
+-----
+
+====
+
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the next section.
+
+
+== Configuring extra LDAP parameters
+
+If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a yaml file that comprises them in your `ldap_extra_settings` dictionary.
+
+.Procedure
+
+. Create a yaml file that will contain `ldap_extra_settings`, such as the following:
++
+----
+#ldapextras.yml   
+---
+ldap_extra_settings:
+  AUTH_LDAP_USER_ATTR_MAP: '{"first_name": "givenName", "last_name": "sn", "email": "mail"}'
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
+
+. Use this example to set up a superuser flag based on membership in an LDAP group.
```

Comment:
This seems to be an optional step, so consider formatting it as "Optional: Set up superuser access based on membership type in an LDAP group" This is a global comment, so please make necessary changes throughout this procedure. Reference: https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=Optional+and+conditional+steps

Reply:
Stet. All of the steps under the heading "Configuring extra LDAP parameters" are just extra configurations that a customer can do. They are all similar. To add optional to just one of those steps would be inconsistent. So this will remain.